apps/web/src/server/views/applications/case/layouts/applications-case-layout.njk

```diff
@@ -41,13 +41,13 @@
 {% endblock %}
 
 {% block pageContent %}
-	<div class="govuk-grid-row">
+	<div class="govuk-grid-row display--flex">
 		{% block sectionMenu %}
-			<div class="govuk-grid-column-one-third">
+			<div class="pins-list-menu-container">
```

Comment:
side menu of case pages (all of them), was shrinked. From 1/3 of the page (standard govuk width), to only 220px.

this is one of the reason of the many updated snapshots